app/views/Dashboard/Country/index.tsx

```diff
@@ -134,12 +167,20 @@ function Country(props: Props) {
 
     const countryVariables = useMemo((): CountryQueryVariables => ({
         iso3: filterValues?.country ?? 'AFG',
+        gte: '2021-08-01',
```

Comment:
why this?

Reply:
Let's use the current date and calculate.